### PR TITLE
[PLAT-430] Fix circular dependency by moving AgencyStream to the mds-ingest-service

### PR DESCRIPTION
**Pull request by avatarneil**
## 📚 Purpose
Fixes a circular dependency between mds-stream and mds-ingest-service.

## 👌 Resolves:
`WARN  There are cyclic workspace dependencies`

## 📦 Impacts:
See changesets

### CODE REVIEW

File: packages/mds-ingest-service/stream/index.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+const writeTripMetadata = async (metadata: TripMetadata) => {
+  return Promise.all([
+    ...(env.NATS ? [IngestStreamNats.writeTripMetadata(metadata)] : []),
+    ...(env.KAFKA_HOST ? [IngestStreamKafka.writeTripMetadata(metadata)] : [])
+  ])
+}
+
+export const IngestStream = {
```
Comment (drtyh2o) [inline packages/mds-ingest-service/stream/index.ts:105]: nit: Why not just have `IngestStream` wrap the actual functions?

**Review by drtyh2o — APPROVED** [review]
Nice!

**Review by PeterSP — APPROVED** [review]

File: packages/mds-ingest-service/stream/index.ts (hunk 1 above)
Comment (drtyh2o) [reply packages/mds-ingest-service/stream/index.ts:105]: The former.

Comment (avatarneil) [reply packages/mds-ingest-service/stream/index.ts:105]: No particular reason!

**Pull request merged**